__tests__/integration/previewRDFHelper.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import pupExpect from 'expect-puppeteer'
+
+export async function previewRDFSetup() {
```

Comment:
Can this be renamed to `fillInRequredFieldsForBibframeInstance`?

Reply:
Done.